### DISCUSSION TITLE
Icon Default 256x256

### DISCUSSION THREAD

**Discussion by Mojas84** [genesistoxical/drop-icons#1] [Q&A] [answered]
Hi please would it be convert to 256x256 default... Must delete smaller image in Gimp......

**Comment by genesistoxical** [comment]
Hello, currently the icons generated by `Drop Icons` contain 6 different sizes to have a better visualization.

If we tried to make it **256x256** by default, the other sizes would not be added, so the icon would look good in the normal or large thumbnail, but in the **details view** it would look pixelated, as shown in the image.

If you don't have a problem with the icon showing pixelated in the **details view**, I can implement the option you need in version 2.0.1 (I'm also sharing a screenshot of what the **256x256** icon looks like in Gimp).

> I hope I have understood your request (I speak Spanish) so if not, please let me know. :)

![ico-256-example](https://github.com/genesistoxical/drop-icons/assets/27597180/ea1db23d-eddf-44f4-be6b-b56b984f3ffd)

**Comment by genesistoxical** [comment, marked as answer]
Version [2.1.1](https://github.com/genesistoxical/drop-icons/releases/tag/2.1.1) has already been released, which includes the improvement you requested. ✅
